yarn.lock

```diff
@@ -8468,22 +8468,10 @@ eth-ens-namehash@^2.0.8:
     idna-uts46-hx "^2.3.1"
     js-sha3 "^0.5.7"
 
-eth-hd-keyring@^2.0.0:
-  version "2.0.0"
-  resolved "https://codeload.github.com/brave/eth-hd-keyring/tar.gz/bff92be9c3129bf45734d751b1c3abc73aa7a0f9"
-  dependencies:
-    brave-crypto "^0.2.1"
-    eth-sig-util "^2.2.0"
-    ethereumjs-util "^6.1.0"
-    ethereumjs-wallet "^0.6.3"
-    tweetnacl "^1.0.1"
-    xtend "^4.0.1"
-
-eth-hd-keyring@brave/eth-hd-keyring:
+eth-hd-keyring@^2.0.0, eth-hd-keyring@brave/eth-hd-keyring:
   version "2.0.0"
   resolved "https://codeload.github.com/brave/eth-hd-keyring/tar.gz/b6cf593fc77572a313f101b1e691a68b333a63dd"
   dependencies:
-    bip39 "^3.0.2"
```

Comment:
not so sure about why this is here since `eth-hd-keyring@ b6cf593fc77572a313f101b1e691a68b333a63dd` does require bip39; maybe because keyringcontroller already has a bip39 dep. let's try it and see.